### ISSUE TITLE
Amilia

### ISSUE THREAD

**Issue by beenlouis** [PipedreamHQ/pipedream#4038]
**Name of app / service**
Amilia

**Link to developer documentation**
https://www.amilia.com/apidocs

**Is lack of support preventing you from building workflows, or do you have a workaround?**
It is definitely challenging to build workflows manually.

**Are there specific actions, or triggers, you'd like to see for this app? Please let us know here or use the Action and Trigger issue templates to open requests for each!**
I want to connect program accounts, activities, and changes with other services (specifically our CRM).

**Comment by sergioeliot2039** [comment]
Hi @beenlouis , thanks for the request!

In order to create an account, a new user needs to schedule a demo. I have requested them one and have booked a demo.

Perhaps for velocity, if you are up to, can you share your username/password, you will have to change them once the integration is done. 

> In order to acquire the JWT-Token you need to use the Amilia authentication endpoint and supply a valid username and password. The username and password should combined with a single colon (:) and passed in the header in the following format:

For privacy sake, you may join our Pipedream-Users public space at Slack https://join.slack.com/t/pipedream-users/shared_invite/zt-11sazmsg0-oS2VW~mZfhtk3XfEEvb~JA

you can DM me directly, I go by Serge (Sergio Rodriguez) or share to my email sergio at pipekit.com

**Comment by sergioeliot2039** [comment]
fyi i booked for a demo on august 30th in the morning

**Comment by sergioeliot2039** [comment]
Hi @beenlouis, a basic frame of integration for Amilia app has been integrated -- https://pipedream.com/apps/amilia

**Comment by sergioeliot2039** [comment]
@beenlouis fyi I created a new issue to track component development -- https://github.com/PipedreamHQ/pipedream/issues/4056
feel free to track progress there, if you'd like
